### ISSUE TITLE
Colorbar and slider broken

### ISSUE THREAD

**Issue by LucaMarconato** [scverse/napari-spatialdata#227]
The colorbar min max is not updated and the slider doesn't perform any action. 

I think having the slider and the colorbar in the same widget would actually be better, maybe we should just replace them with a different widget, or remove them altogether.

**Comment by melonora** [comment]
Yeah I am aware. As far as I know this was already broken in early versions. I take it that this is supposed to clip numerical vectors when coloring? @giovp
